composer.json

```diff
@@ -24,14 +24,13 @@
         "symfony/event-dispatcher": "^4.4 || ^5.0 || ^6.0",
         "symfony/translation": "^4.4 || ^5.0 || ^6.0",
         "symfony/yaml": "^4.4 || ^5.0 || ^6.0",
-        "psr/container": "^1.0"
+        "psr/container": "^1.0 || ^2.0"
     },
 
     "require-dev": {
         "symfony/process": "^4.4 || ^5.0 || ^6.0",
         "phpunit/phpunit": "^8.5 || ^9.0",
         "herrera-io/box": "~1.6.1",
-        "container-interop/container-interop": "^1.2",
```

Comment:
What do you think about adding this library to `conflict` section? This way any project that still depends on this library will be unable to install a newer version of Behat that removes support for it - and they won't be shocked by Behat breaking randomly after upgrading it.

Reply:
:+1: Great idea :)